### ISSUE TITLE
[FEATURE REQUEST] [TIMELINE] Add indicator to differ between item and perk damage timeline

### ISSUE THREAD

**Issue by DarkIntaqt** [RiotGames/developer-relations#698]
Currently, it is not possible to determine if the damage dealt in a "CHAMPION_KILL" timeline event is by a passive item or by a perk. 
I would like to see which "thing" dealt the damage, especially as it is possible to see in the client. As you can see in the attached image below, it is not possible to see if the damage was dealt by an item or by a perk (when not comparing to a screenshot of course). 

![](https://cdn.discordapp.com/attachments/1011040493569851463/1029439290700201984/timeline.png)

You could add something like this
```json
{
   "basic": false,
   "magicDamage": 0,
   "name": "CHAMPIONNAME",
   "participantId": 1,
   "physicalDamage": 0,
   "spellName": " (maybe add the perk or item name here)",
   "spellSlot": -1 ,
   "trueDamage": 0,
   "type": "ITEM || PERK",
}
```

**Comment by bayu01** [comment]
Would love to be able to get to this data. Currently there is no other API that I know of that would allow us to get item data

**Comment by benjaminbours** [comment]
I completely agree, I just build an [online death recap](https://www.deathrecap.lol) based on these data and the first feedback I have received is the fact there are a lot of unknown source of damage, due to the empty `spellName` property in the data structure.

Its been almost one year it has been reported, do we have any update about it?

Can we do anything to solve this issue? I'm willing to help

**Comment by benjaminbours** [comment]
Hello @tisbells, sorry to bother you but can you please provide any update about this issue or at least acknowledge the issue and let us know if it is considered or not?

Thanks in advance,

Benjamin
